packages/firebase_dynamic_links/lib/src/firebase_dynamic_links.dart

```diff
@@ -104,3 +142,13 @@ class PendingDynamicLinkDataIOS {
   /// declares higher [minimumVersion] than currently installed.
   final String minimumVersion;
 }
+
+class OnLinkErrorException {
```

Comment:
I think you may have changed this back to `OnLinkErrorException`.

Reply:
What you mean? It's already `OnLinkErrorException`

Reply:
In the comment here: https://github.com/flutter/plugins/pull/1687#discussion_r294047856, you said you changed it to `OnLinkErrorData`, but here its name is `OnLinkErrorException`.

I might be looking at the wrong commits tho.

Reply:
Ok :)